lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -30,6 +30,18 @@
       - If C(advertise_addr) is not specified, it will be automatically
           detected when possible.
     type: str
+  default_addr_pool:
+    description:
+      - Default address pool in CIDR format.
+      - Requires API version >= 1.39.
```

Comment:
I think we should mention here (and to the other options which can only be specified on swarm init) that these are not considered for idempotency checking since they can not be updated.